hrm-domain/hrm-core/contact/contactService.ts

```diff
@@ -246,8 +262,12 @@ export const patchContact = async (
         });
       }
     }
+    const res = await initProfile(conn, accountSid, restOfPatch);
+    if (isErr(res)) {
+      throw res.rawError;
+    }
 
-    const { profileId, identifierId } = await initProfile(conn, accountSid, restOfPatch);
+    const { profileId, identifierId } = res.data;
```

Comment:
Here's where `res.unwrap` might come handy, but as a question so I know when to use it and when not: why not using the `ensureRejection` wrapper?